docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     command: python manage.py runserver 0.0.0.0:8000
     volumes:
       - ./app/eventyay:/usr/src/app/eventyay
+      - /usr/src/app/eventyay/webapp/dist
+      - /usr/src/app/eventyay/webapp/node_modules
```

Comment:
What shoud this do? That doesn't make sense, and is incorrectly formatted.
Please remove the two lines.